aiida/cmdline/commands/cmd_import.py

```diff
@@ -44,6 +91,7 @@ def _try_import(migration_performed, file_to_import, archive, group, migration,
     :param group: AiiDA Group into which the import will be associated.
     :param migration: Whether or not to force migration of archive, if needed.
     :param non_interactive: Whether or not the user should be asked for input for any reason.
+    :param more_archives: Whether or not there are more archives to be imported.
```

Comment:
I'm not this function needs to know about other archives.
I think this would better be handled outside this function.

Reply:
It seems to only be used to transfer the knowledge on to `_echo_error`. I could tuck it into `kwargs`? But it is _needed_ for the `_echo_error` function to determine what next action to take...